README.md

```diff
@@ -13,13 +13,17 @@ Run `make`.
 
 The Source connector subscribes to Salesforce platform events and queries published events in real time.
 
+## Destination
+
+The Destination connector subscribes to Salesforce platform events and publishes incoming records to the event in real time.
```

Comment:
I might be missing something, but does the destination connector **subscribe** for real? I would have expected it to publish to something (whatever that thing in Salesforce is called). It's a bit contradictory to the latter part of the sentence: it subscribes and publishes.

Reply:
Thats true wording is confusing, it doesn't necessarily subscribe (it checks if topic exists / looks at topic schema) 

I'll reword this to: 

"The Destination connector publishes incoming records to Salesforce platform events in real time."

Reply:
Great, thanks for the change!